token/core/zkatdlog/nogh/auditor.go

```diff
@@ -14,29 +15,42 @@ import (
 	api3 "github.com/hyperledger-labs/fabric-token-sdk/token/driver"
 )
 
+// AuditorCheck verifies if the passed tokenRequest matches the tokenRequestMetadata
 func (s *Service) AuditorCheck(tokenRequest *api3.TokenRequest, tokenRequestMetadata *api3.TokenRequestMetadata, txID string) error {
 	logger.Debugf("check token request validity...")
+	if s.TokenCommitmentLoader == nil {
```

Comment:
no need for this check evey time. This is set at service construction time.

Reply:
OK